tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -504,7 +504,8 @@ public TestSetup apply(Long aLong) throws Throwable {
 
           @Override
           public void onSubscribe(final Subscription subscription) {
-            env.debug(String.format("whiteboxSubscriber::onSubscribe(%s)", subscription));
+            if (env.debugEnabled())
```

Comment:
Needs curly braces to adhere to codebase style.

Reply:
@JakeWharton Good point—addressed!